### ISSUE TITLE
Hover over shortcut tooltip - is it working ?

### ISSUE THREAD

**Issue by GaryRoussak** [tjackenpacken/taskbar-groups#269]
Hi,

There's a tooltip appearing over my Taskbar Group when I add it to the task bar.

However, the release notes (v0.2.0.0) seem to suggest I should also be seeing tooltips when hovering over the individual shortcuts themselves (inside the group). They aren't happening, so I don't understand the purpose of being able to edit them.

Could something have happened to my Windows 11 configuration ? I've "re-installed" v0.2.0.0 but still got the same issue.

Suggestions ?

Thanks.

![Screenshot_20230210_160728](https://user-images.githubusercontent.com/30197668/218146816-79454220-c0b2-468a-a19a-d975a7f9238f.png)

**Comment by gaveitatry2** [comment]
I downloaded TaskbarGroups v0.2.0.0 today on Feb 10, 2023 for Windows 10. I wanted to use this program, but unfortunately I could not. I tested the program out by adding 11 shortcuts. Each shortcut was to a Ren'Py Visual Novel game. And every shortcut had the same icon. When I pinned the taskbar group to the taskbar, I was sad to see that I could not distinguish any of the 11 shortcuts from each other because there was no tool tip to tell me the shortcut name. The only workaround I can see for this is to create custom icons for each shortcut but that's too much work and headache. Hopefully a future version of this program will have shortcut program names on tooltips when you hover over the shortcuts.

**Comment by aegor** [comment]
Yes, Hower Over App Icon Is the Killer Feature, and Without this is very difficult to use this application
